packages/plugin-commands-installation/src/recursive.ts

```diff
@@ -289,7 +289,17 @@ export default async function recursive (
   const limitInstallation = pLimit(opts.workspaceConcurrency ?? 4)
   await Promise.all(pkgPaths.map(async (rootDir: string) =>
     limitInstallation(async () => {
-      const hooks = opts.ignorePnpmfile ? {} : requireHooks(rootDir, opts)
+      const pnpmfileHooks = opts.ignorePnpmfile ? {} : requireHooks(rootDir, opts)
```

Comment:
let's also change what the requireHooks return. Let's return an array of hooks.

Reply:
I did that at first but the return type of requireHooks -> CookedHooks is used in config for hooks definition so change there have impact on multiple places in code. Do you suggest spiting the type (separate Hooks for config and requireHooks result) or changing to array everywhere?

Reply:
ok, I'll look into it.